welcometothemoon.js

```diff
@@ -656,6 +656,15 @@ define([
         this.launchActionOnSlotClick(args.slots, 'actCirclePlant');
       },
 
+      onEnteringStateStirWaterTanks(args) {
+        this.onClick(`slot-${this.player_id}-${args.slots[0]}`, () => {
```

Comment:
I think i would probably just go with singular string instead of array here

Reply:
My frustration is high as I've tried that while implementing and it didn't work. But now it does...